drivers/net/bnxt/bnxt_ethdev.c

```diff
@@ -819,6 +820,78 @@ static int bnxt_rss_hash_update_op(struct rte_eth_dev *eth_dev,
 	return 0;
 }
 
+int rte_pmd_bnxt_set_tx_loopback(uint8_t port, uint8_t on)
```

Comment:
I suggest moving all these rte_pmd_bnxt_xyz() implementions into its own rte_pmd_bnxt.c 
Are you talking of supporting in bnxt_hwrm.c or HWRM API?
I think HWRM API already allows toggling NO-VEB, VEB, VEPA anyway.
hwrm_func_cfg

Am I missing something?